examples/Example1.cpp

```diff
@@ -5,16 +5,26 @@
 #include <Arduino.h>
 #include <ABBAurora.h>
 
+
+
 #define RX2 16
 #define TX2 17
+#define RX1 3
+#define TX1 1
 #define INVERTER_ADDRESS 2
 #define TX_CONTROL_GPIO 4
+#define TX_CONTROL_GPIO2 16
 
 ABBAurora *inverter;
 void setup()
 {
     Serial.begin(115200);
+    
+#if defined(ESP832)
```

Comment:
If comment for line 12 is possible, we might get rid of this condition here since TX/RX would be correctly defined based on platform.